Stylesheets/Components/avatar.css

```diff
@@ -0,0 +1,45 @@
+.avatar-circle {
+    max-width: 100%;
+    height: auto;
+    border-radius: 50%;
+    width: 2.5rem;
+}
+
+.avatar-square {
+    max-width: 100%;
+    height: auto;
+    border-radius: 10%;
+    width: 2.5rem;
+}
+
+.avatar img {
+    height: 2.5rem;
+    width: 2.5rem;
+}
+
+.avatar-shadow {
+    box-shadow: 10px 10px 2px 10px #888888;
+}
+
+.avatar-sm {
+    width: 1.5rem;
+    height: auto;
+
```

Comment:
format your code so that these extra spaces will be gone

Reply:
sure will use prettier